ethers-core/src/utils/geth.rs

```diff
@@ -21,7 +21,7 @@ const GETH_STARTUP_TIMEOUT_MILLIS: u64 = 10_000;
 const GETH_DIAL_LOOP_TIMEOUT: Duration = Duration::new(20, 0);
 
 /// The exposed APIs
-const API: &str = "eth,net,web3,txpool,admin,personal,miner";
+const API: &str = "eth,net,web3,txpool,personal,miner";
```

Comment:
@mattsse I forgot to remove the default admin here. This would change the default behaviour.

Reply:
oh my bad didn't realise this was in by default,

sorry about the confusion.

we want to keep admin by default. so the enable_admin should be removed, but we can keep the Vec, so custom namespaces can be enabled.